extensions/Vendor/BLENDER_physics/README.md

```diff
@@ -0,0 +1,182 @@
+# BLENDER\_physics
+
+## Contributors
+
+* Mitchell Stokes, Blender, <mailto:mogurijin@gmail.com>
+* Daniel Stokes, Blender, <mailto:kupomail@gmail.com>
+* Steven Vergenz, AltspaceVR, <mailto:steven@altvr.com>
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against glTF 1.0 and 2.0 specs.
+The only difference for this extension for 1.0 vs 2.0 is the `mesh-id`.
+
+## Overview
+
+This extension extends nodes to include information necessary to construct rigid body objects for a physics engine (e.g., [Bullet](https://bulletphysics.org/)).
+
+## glTF Schema Updates
+
+A `BLENDER_physics` object is added to the extensions list of any `node` that should participate in the physics simulation.
+The properties available are listed in the table below.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**collisionShape**|`string`|The shape a physics simulation should use to represent the node|No, default: `BOX`|
+|**mass**|`number`|The 'weight', irrespective of gravity, of the node|No, default: `1.0`|
+|**static**|`boolean`|Whether or not the Node should be moved by physics simulations|No, default: `false`|
+|**boundingBox**|`array`|The dimensions of the local (i.e., does not include the node's transform) bounding box of the collision shape centered on the origin|Yes|
```

Comment:
I take it this gives the dimensions on the XYZ axes? Perhaps "size" or "scale" would be a better name for this?